contributed_definitions/nyaml/NXammeter.yaml

```diff
@@ -0,0 +1,11 @@
+category: base
```

Comment:
Why does this need to be its own base class? Can you not just use `amperemeter(NXsensor)` when you want to describe an ammeter in the app def?

Reply:
Changed via adding suggestion in NXem how to name and where to place instances of sensors and actuators.